source/docs/03-metadata.xml

```diff
@@ -1444,6 +1444,82 @@
                      <p>Watermark</p>
                      <p>Stamp</p>
                      <p>physMedium</p>-->
+                     <p>Encoding the exent and the dimensions of a source:</p>
+                     <p><specList>
+                        <specDesc key="extent"/>
+                     </specList></p>
+                     <p>The element <gi scheme="MEI">extent</gi> is used to express size in terms other than physical dimensions. This can be done in units such as ‘number of pages’, ‘number of folios’, ‘records’, ‘bytes’, ‘physical components’, etc. For specifying the physical size of the material, for example, height and width, the use of the <gi scheme="MEI">dimensions</gi> element is recommended.</p>
+                     <p>
+                        <figure>
+                           <head/>
+                           <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample085.txt" parse="text"/></egXML>
+                        </figure>
+                     </p>
+                     <p><specList>
+                        <specDesc key="dimensions"/>
+                     </specList></p>
+                     <p>The <gi scheme="MEI">dimensions</gi> contain information about the physical size of a source or entity in general. Usually the dimensions are represented by numerical data. The elements <gi scheme="MEI">height</gi>, <gi scheme="MEI">width</gi>, <gi scheme="MEI">depth</gi>, and <gi scheme="MEI">dim</gi> are available for circumstances that require the capture of the individual dimensions of an object. Do not confuse this element with the <gi scheme="MEI">extent</gi> element, which is used to indicate the quantity of described materials.</p>
```

Comment:
```suggestion
                     <p>The <gi scheme="MEI">dimensions</gi> contain information about the physical size of a source. Usually the dimensions are represented by numerical data. The elements <gi scheme="MEI">height</gi>, <gi scheme="MEI">width</gi>, <gi scheme="MEI">depth</gi>, and <gi scheme="MEI">dim</gi> are available for circumstances that require the capture of individual dimensions.</p>
```

I removed the last sentence because unless you're going to provide an example of how it might be confused, you're only inviting the user to be confused by not knowing how they can be confused by it!

Reply:
Maybe something like "To indicate the quantity of the described materials, please refer to the extent element" ...?

Reply:
Concerning the first deletion: I tried to explain that `<dimensions>` is not limited to a (musical) source, because it can also occur, e.g., at `<watermark>`.

Concerning the second: I see, but I still think it is important to point out the difference between dimensions and extent. 

```suggestion
                     <p>The <gi scheme="MEI">dimensions</gi> contain information about the physical size of a source. Usually the dimensions are represented by numerical data. The elements <gi scheme="MEI">height</gi>, <gi scheme="MEI">width</gi>, <gi scheme="MEI">depth</gi>, and <gi scheme="MEI">dim</gi> are available for circumstances that require the capture of individual dimensions. To indicate the quantity of the described materials, please refer to the extent element.</p>
```